ansible-ipi-install/roles/installer/tasks/50_extramanifests.yml

```diff
@@ -14,6 +14,15 @@
     - release_version.split('.')[0]|int == 4 and release_version.split('.')[1]|int < 8
   delegate_to: localhost
 
+- name: Copy compact cluster related manifest
+  copy:
+     src: cluster-scheduler-02-config.yml
+     dest: "{{ extramanifestsopenshift_path }}/"
+     mode: 0644
+  when:
+    - compact_cluster|bool
```

Comment:
You have a change to this file and an additional validation in ansible-ipi-install/roles/node-prep/tasks/10_validation.yml as part of PR #222. Should those changes be pulled back to this PR for feature separation hygiene?